### ISSUE TITLE
won't launch

### ISSUE THREAD

**Issue by B-Nerd** [monokuma9/dolphin-triforce-linux#2]
when i tried to launch this i got this error dolphin-emu: symbol lookup error: /usr/lib/libtiff.so.6: undefined symbol: jpeg12_write_raw_data, version LIBJPEG_8.0

**Comment by monokuma9** [comment]
Yeahhhh this is one of the reasons I eventually just gave up with this (not to mention this repo literally existed as a link to host for the AUR).

It was only by lucky chance that this ever worked at all tbh; I happened to stumble on the exact right script that just so happened to work (and credited the creator in the related forum posts about it, I’m no credit hog I just put this here as a host for the AUR). The user has since deleted the script from the internet, which I found while trying to patch the broken AppImage. And after that, I basically gave up.

I apologize that I can’t help much here, even if I was working on this still, Id have to piece the script together from scratch with no real reference to the original (unless the script happens to be deep on some drive that Im not aware of)

**Comment by B-Nerd** [comment]
but there is no aur release anymore
![Screenshot_20240225_015248](https://github.com/monokuma9/dolphin-triforce-linux/assets/65131530/89b9856b-642a-4441-b5a5-1d0f4d2235ed)

**Comment by monokuma9** [comment]
Correct, I orphaned that too after realizing that my time is too limited to tackle it anymore. You’re welcome to take a stab though, don’t let me giving up dissuade you

Anti-Ultimate is the user you’re looking for btw. Despite having never built a Triforce build themselves they know more about this stuff than I do, as it was their script that saved me here.

**Comment by monokuma9** [comment]
I never circled back here, literally forgot about it tbh

**Comment by B-Nerd** [comment]
ah oke no problem i wish Anti-Ultimate page did not have dead links

**Comment by pilgrimlvii** [comment]
With this command it works on Xubuntu 24.04.2

`LD_PRELOAD="/lib/x86_64-linux-gnu/libselinux.so.1:/lib/x86_64-linux-gnu/libgmodule-2.0.so.0:/lib/x86_64-linux-gnu/libdbus-1.so.3:/lib/x86_64-linux-gnu/libatk-1.0.so.0" ./dolphin-emu-triforce-4.0-315.glibc2.15-x86_64.AppImage`

![Image](https://github.com/user-attachments/assets/f8367643-f3ad-492a-831e-370f60b10342)
